synadm/cli/__init__.py

```diff
@@ -62,6 +71,7 @@ class APIHelper:
     FORMATTERS = {
         "pprint": pprint.pformat,
         "json": json_pretty,
+        "minified": json.dumps,
```

Comment:
This isn't so... minified

```json
{"server_version": "1.76.0 (b=HEAD,t=v1.76.0,eafdb12,dirty)", "python_version": "3.10.10"}
```

there are spaces in between the `:` and `,` which could be removed. it might not be practically possible to make that happen because of how the formatter works (likely needs a wrapper function).

see the compact encoding section at https://docs.python.org/3/library/json.html

Reply:
Hm ok, well I was actually just looking for a better name that isn't too long for the "plain" json output format. I'd like to express that this is the json the api would respond "plain text, non-formatted", since  `json` is now used for the formatted json.

Hm, that puts me back to the start but well, yes, let's brainstorm again: Any suggestions how this output mode could be called? `onelinejson``, but that's loooon and ugly as well, haha ;-)

Reply:
or just `oneline` ? The help would tell the use what it means anyway. What do you think?

(I don't want to fiddle around with really "minifying" at the moment, but if you would be up for it I won't stop you :-)

Reply:
Oh, I should have read your suggestion before blablaing. I'm sorry! I read the "compact encoding" section. So if we would do that in a formatting_function, would you say we could call it `minified` ? If yes, I would just implement that. I like the idea!

Reply:
That is changed now. Actually I love the idea! Many thanks for that great suggestion! We now have a "real" minified mode slash scripting- slash performance-optimized mode. Neat!